ofrak_core/ofrak/ofrak_cli.py

```diff
@@ -275,11 +285,102 @@ def handler(ofrak_env: OFRAKEnvironment, args: Namespace):
             print(dependency_info)
 
 
+class GUISubCommand(OFRAKSubCommand):
+    def create_parser(self, ofrak_subparsers):
+        gui_parser = ofrak_subparsers.add_parser(
+            "gui",
+            help="Launch the OFRAK GUI server.",
+            description="Launch the OFRAK GUI server.",
+            formatter_class=ArgumentDefaultsHelpFormatter,
+        )
+        gui_parser.add_argument(
+            "-H",
+            "--hostname",
+            action="store",
+            help="Set GUI server host address.",
+            default="127.0.0.1",
+        )
+        gui_parser.add_argument(
+            "-p",
+            "--port",
+            action="store",
+            type=int,
+            help="Set GUI server host port.",
+            default=8080,
+        )
+        gui_parser.add_argument(
+            "-b",
+            "--backend",
+            action="store",
+            help="Set GUI server backend.",
+            default=None,
+        )
+        gui_parser.add_argument(
+            "-v",
+            "--verbose",
+            action="store_true",
+            help="Enable verbose mode for debugging",
+            default=None,
+        )
+        gui_parser.add_argument(
+            "-q",
+            "--quiet",
+            action="store_true",
+            help="Enable quiet mode to minimize logging",
+            default=None,
+        )
+        return gui_parser
+
+    @staticmethod  # pragma: no cover
+    def handler(ofrak_env: OFRAKEnvironment, args: Namespace):
+        if args.verbose is True:
+            ofrak = server.OFRAK(server.logging.DEBUG)
+
+        elif args.quiet is True:
+            ofrak = server.OFRAK(server.logging.WARNING)
+
+        else:
+            ofrak = server.OFRAK(server.logging.INFO)
+
+        if args.backend is not None:
+            if args.backend.lower() == "binary-ninja":
+                import ofrak_capstone  # type: ignore
+                import ofrak_binary_ninja  # type: ignore
+
+                ofrak.injector.discover(ofrak_capstone)
+                ofrak.injector.discover(ofrak_binary_ninja)
+
+            elif args.backend.lower() == "ghidra":
+                import ofrak_ghidra  # type: ignore
+
+                ofrak.injector.discover(ofrak_ghidra)
+
+            elif args.backend.lower() == "angr":
+                import ofrak_angr  # type: ignore
```

Comment:
This needs to import and discover `ofrak_capstone` as mentioned in the other case.